docs/chapters/traversal/index.md

```diff
@@ -0,0 +1,63 @@
+# Understand Jina Recursive Document Representation
+
+In Jina, each `Document` is represented as a rooted binary-tree:
```

Comment:
This is personally a little missleafing to me. It seems that a Document has only one chunk and one match.

Reply:
And I also do not think is accurate.